Product_Insights/Twitter/process_twitter_data.py

```diff
@@ -0,0 +1,215 @@
+import datetime
+
+import pandas as pd
+
+from google.cloud import bigquery
+from google.cloud import storage
+from google.cloud.exceptions import Forbidden, NotFound
+
+from Product_Insights.Classification.utils \
+        import keywords_based_classifier
+from Product_Insights.Classification.create_classification_table \
+        import create_keywords_map
+from Product_Insights.Classification.upload_keywords_map \
+        import upload_keywords_map
+from Product_Insights.Sentiment.utils \
+        import gc_detect_language, gc_sentiment, discretize_sentiment
+from Product_Insights.Twitter.create_twitter_tables \
+        import create_twitter_sentiment
+
+PROJECT_ID = 'marketing-1003'
+
+INPUT_DATASET = 'sumo_views'
```

Comment:
Please remove lines 22 to 28 since these variables are passed in from entry point.